.github/config.yml

```diff
@@ -15,10 +15,14 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-   &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
+  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
   \
   \
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/welcome/Layer5-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery-operator/master/.github/welcome/meshery-celebration-light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery-operator/master/.github/welcome/meshery-celebration-dark.svg">
+  <img alt="Meshery Logo" src="https://raw.githubusercontent.com/meshery/meshery-operator/master/.github/welcome/meshery-celebration-dark.svg">
```

Comment:
I cannot access this picture https://raw.githubusercontent.com/meshery/meshery-operator/master/.github/welcome/meshery-celebration-dark.svg. Is this right?

Reply:
Yes, this link is correct. It is not working currently because I have given the link of the this repository and in this repo the meshery-celebration-dark image is not present because this pull request is not yet merged so it will not work. 
If u merge this pr then it will work.
You can try a demo by replacing meshery-celebration-dark.svg by  Layer5-celebration.png (https://raw.githubusercontent.com/meshery/meshery-operator/master/.github/welcome/Layer5-celebration.png)  then it will work as Layer5 image is present in .github/welcome folder in this meshery-operator repo.

This pr will add the 2 images meshery-celebration-dark and meshery-celebration-light to the .github/welcome folder so this link will work.

Reply:
Please update the code to the latest `main` branch. We can merge this one.